### ISSUE TITLE
bug: Gradience won't start

### ISSUE THREAD

**Issue by zt64** [GradienceTeam/Gradience#746]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Gradience GUI won't launch. Only CLI works.

### To Reproduce

1. Launch terminal
2. Execute `gradience` command

### Expected behavior

The gradience UI should show up

### Screenshots

![image](https://user-images.githubusercontent.com/31907977/228726028-ecd5dd22-2ebf-4ac0-a7fd-d88c8dd5df3b.png)

### OS

Arch Linux

### DE/WM version

Cinnamon 5.6.8

### Version

0.4.0-1, `main` brance

### Installation method

AUR

### Additional context

Used to work fine but then something changed and now it does this

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tfuxu** [comment]
@zt64 
Try running Gradience with GTK debugging enabled: `GTK_DEBUG=all gradience`

**Comment by zt64** [comment]
> GTK_DEBUG=all gradience

Ok this shows the debug window and I get logs.

```(process:177230): Gtk-WARNING **: 17:15:55.897: GtkImage 0x55d3f6fd7f40 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f6fd87c0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f729a5d0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f729ad10 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72a66b0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72a6e40 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72a9d70 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72aa500 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72ad460 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72adbf0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72b1170 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72b1b80 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72c5240 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72c5c50 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72c9860 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72ca270 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72cf7c0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.

(process:177230): Gtk-WARNING **: 17:15:55.898: GtkImage 0x55d3f72d01d0 reported baselines of minimum -2147483648 and natural -2147483648, but sizes of minimum 16 and natural 16. Baselines must be inside the widget size.
[Gradience] WARNING: Custom preset not found. Fallback to Adwaita
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 252, in load_preset_from_css
    variables, palette, custom_css = parse_css(
  File "/usr/lib/python3.10/site-packages/gradience/backend/css_parser.py", line 47, in parse_css
    with open(path, "r", encoding="utf-8") as sheet:
FileNotFoundError: [Errno 2] No such file or directory: '/home/nick/.config/gtk-4.0/gtk.css'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 148, in do_activate
    self.load_preset_from_css()
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 276, in load_preset_from_css
    self.load_preset_from_resource(
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 307, in load_preset_from_resource
    self.load_preset_variables_from_preset()
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 333, in load_preset_variables_from_preset
    self.reload_variables()
  File "/usr/lib/python3.10/site-packages/gradience/frontend/main.py", line 440, in reload_variables
    css_provider.load_from_data(gtk_css.encode())
TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)
```

**Comment by zt64** [comment]
Okay, I made `/home/nick/.config/gtk-4.0/gtk.css` and switched to the git version from the aur and it finally launches

**Comment by tfuxu** [comment]
You probably hasn't been able to launch Gradience because of that `TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)` error. We fixed it in 0.4.1 bugfix, but for some reason the package for Gradience in AUR hasn't been updated yet. 

@0xMRTT could you update it?
